linera-views/src/reentrant_collection_view.rs

```diff
@@ -272,6 +272,21 @@ where
     pub async fn try_load_entry(
         &self,
         short_key: Vec<u8>,
+    ) -> Result<ReadGuardedView<W>, ViewError> {
+        {
+            let mut hash = self.hash.lock().await;
+            *hash = None;
+        }
+        self.local_try_load_entry(short_key).await
+    }
+
+    /// When computing the hash we are locking the hash.
```

Comment:
nit: We auto-wrap at 90 (see file .editorconfig and how to install it here: https://github.com/editorconfig/editorconfig-emacs#readme)